docs/index.md

```diff
@@ -1,54 +1,17 @@
-### What Are We Fighting For?
```

Comment:
https://www.youtube.com/watch?v=9OVv-J-LXQU

That, and I associate "fighting" with something related to politics or cancer. It needs to refer to the mechanics of D&D. "What are we doing in D&D combat" is the message people need to get from the first header.